src/praisonai/praisonai/cli/commands/run.py

```
@@ -55,6 +56,18 @@ def run_main(
     
+    # Resolve configuration if model not explicitly provided
+    if model is None:
+        try:
+            config = resolve_config()
+            if config.agent.model:
+                model = config.agent.model
+                if verbose:
+                    output.print_info(f"Using model from config: {model}")
+        except Exception:
+            # Silently continue if config resolution fails
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)